microsoft/testsuites/cpu/cpusuite.py

```diff
@@ -51,3 +61,69 @@ class CPUSuite(TestSuite):
     )
     def verify_cpu_hot_plug(self, log: Logger, node: Node) -> None:
         verify_cpu_hot_plug(log, node)
+
+    @TestCaseMetadata(
+        description="""
+            This test will check cpu hotplug with storage workload.
+            The cpu hotplug steps are same as `verify_cpu_hot_plug` test case.
+            """,
+        priority=3,
+        requirement=simple_requirement(
+            min_core_count=32,
+        ),
+    )
+    def verify_cpu_offline_storage_workload(self, log: Logger, node: Node) -> None:
+        # run fio process asynchronously on the node and give it 10 seconds to start
+        fio_storage_workload_script = CustomScriptBuilder(
```

Comment:
It's like a metadata of the script, so, it can be in `__init__` of the test suite. The script won't be copied, until it's called from `node.tools`.